pkgs/development/python-modules/channels_redis/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, redis, msgpack, channels, aioredis, async_generator, pytest
+}:
+buildPythonPackage rec {
+
+  pname = "channels_redis";
```

Comment:
Please normalize `pname` to `channel-redis`.

Reply:
Can I confirm this one?
In this case the package is actually named `channels_redis` (plural with underscore, not dash) consistently in their documentation, homepages, and in dependency lists for all `requirements.txt` files.
Are you sure that we want to alter their convention?

Reply:
I meant `channels-redis`.

And yes we do, there's an ongoing effort to normalize the derivation name in this manner and
the nixpkgs documentation on python packages states that attributes should be normalized similarally.

Reply:
For reference:

See https://nixos.org/nixpkgs/manual/#contributing
> Attribute names in python-packages.nix should be normalized according to PEP 0503. This means that characters should be converted to lowercase and . and _ should be replaced by a single - (foo-bar-baz instead of Foo__Bar.baz ) 

And 
> https://nixos.org/nixpkgs/manual/#sec-package-naming
states that dashes are preferred.

I can't recall where the discussion for python packages is specifically.